CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * [BUGFIX]: Remove unnecessary PersistentVolumeClaim [#1245](https://github.com/grafana/tempo/issues/1245)
 * [BUGFIX] Fixed issue when query-frontend doesn't log request details when request is cancelled [#1136](https://github.com/grafana/tempo/issues/1136) (@adityapwr)
 * [BUGFIX] Fixed panic when using etcd as ring's kvstore [#1260](https://github.com/grafana/tempo/pull/1260) (@mapno)
+* [CHANGE] Copy cortex dependencies into Tempo [#1254](https://github.com/grafana/tempo/pull/1254) (@zalegrala)
```

Comment:
i'm inclined to not list this since it should be a no-op from a user's perspective.

Reply:
Fair enough.

Reply:
Dropped this commit.